README.md

```diff
@@ -40,29 +43,30 @@ import './jestGlobalMocks'; // browser mocks globally available for every test
 {
   "globals": {
     "ts-jest": {
-      "tsConfigFile": "src/tsconfig.spec.json"
-    },
-    "__TRANSFORM_HTML__": true
+      "tsConfig": "<rootDir>/src/tsconfig.spec.json",
+      "stringifyContentPathRegex": "\\.html$"
+    }
   },
   "transform": {
-    "^.+\\.(ts|js|html)$": "<rootDir>/node_modules/jest-preset-angular/preprocessor.js"
+    "\\.(ts|js|html)$": "ts-jest"
   },
   "testMatch": [
-    "**/__tests__/**/*.+(ts|js)?(x)",
-    "**/+(*.)+(spec|test).+(ts|js)?(x)"
-  ],
-  "moduleFileExtensions": [
-    "ts",
-    "js",
-    "html"
+    "**/__tests__/**/*.ts?(x)",
```

Comment:
why these 2 regexes were modified ? The old regexes works

Reply:
As much as I can see there is no reason to change them. This was done by **huafu** in the commit before, and I do not know why, maybe js-files are not supported by ts-jest anymore (which I doubt), but I have not tested it.

Will undo it in the next commit.